mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
So the updated docstring would be something like:
```
    By default, this makes each time point (within each epoch) have
    μ=0, σ=1 for each channel type (``'mag', 'grad', 'eeg'``) independently.
```
And I would have expected something more like:
```
   By default, this makes each channel have zero mean across time,
   and scales all channels of the same type (``'grad', 'mag', 'eeg'``)
   by the same factor to achieve unit variance across all time points
   within all epochs.
```
I'm also a bit surprised this is really needed, given that e.g. in our `cov` code we do simple `dict`-based scaling and it seems to work well enough. Why not use that approach here?